src/single_instruction_test/vm.rs

```diff
@@ -81,6 +76,8 @@ impl<'a> Arbitrary<'a> for VirtualMachine {
             register_pointer_flags |= (is_pointer as u16) << i;
         }
 
+        let heaps = Heaps::from_id(current_frame.heap, u)?;
```

Comment:
Inline this variable.

Reply:
will not work nicely due to `current_frame` moving in constructor.